### PR TITLE
Merge blobfuse2.1.1

### PR DESCRIPTION
**Pull request by jfantinhardesty**
### What type of Pull Request is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

### Describe your changes in brief

Update to the 2.1.1 release of blobfuse2.

### Checklist

- [x] Tested locally
- [ ] Added new dependencies
- [ ] Updated documentation
- [ ] Added tests

### Related Issues

- Related Issue #
- Closes #

### CODE REVIEW

**Review by foodprocessor — APPROVED** [review]
Thanks!

File: azure-pipeline-templates/blobfuse2-ci-template.yml
Comment (foodprocessor) [inline azure-pipeline-templates/blobfuse2-ci-template.yml:1]: Can you delete this file?

File: component/azstorage/azstorage.go [hunk 1]
```diff
@@ -171,7 +171,8 @@ func (az *AzStorage) Start(ctx context.Context) error {
 	log.Debug("Starting azstorage stats collector")
 
 	// This is a workaround right now to disable the input watcher thread which continuously monitors below config to change
-	// Running this thread continuously increases the CPU usage by 5% even when there is no activity on cloudfuse mount path
+	// Running this thread continuously increases the CPU usage by 5% even when there is no activity on blobfuse2 mount path
+	// Lifecycle manager init is commented in the "blobfuse2-cpu-usage" branch. Blobfuse2 imports azcopy from this branch.
```
[the remark this answers is not in the document]
Comment (jfantinhardesty) [reply component/azstorage/azstorage.go]: Yes I deleted these comments.

File: azure-pipeline-templates/blobfuse2-ci-template.yml
Comment (jfantinhardesty) [reply azure-pipeline-templates/blobfuse2-ci-template.yml:1]: Yes.

**Pull request merged**